functions/billing/main.py

```diff
@@ -104,6 +104,9 @@ def __is_billing_enabled(project_name, projects):
     try:
         res = projects.getBillingInfo(name=project_name).execute()
         return res['billingEnabled']
```

Comment:
Would it be cleaner to use `return 'billingEnabled' in res and res['billingEnabled']`?

Reply:
Yes, though I feel like that's slightly harder to understand.

Reply:
I would be alright with keeping it as is. This style is fairly common in Python. "Easier to ask for forgiveness than permission".

https://docs.python.org/3/glossary.html?highlight=eafp

> Easier to ask for forgiveness than permission. This common Python coding style assumes the existence of valid keys or attributes and catches exceptions if the assumption proves false. This clean and fast style is characterized by the presence of many try and except statements. The technique contrasts with the LBYL style common to many other languages such as C.

Reply:
Ah, thanks for clarifying @busunkim96! (I guess Node has taught me to optimize for brevity. 😄 )